uint8         msgcnt
float64       u8x
float64       u8y
float64       u8z